serial/aio.py

```diff
@@ -1,121 +1,127 @@
-#!/usr/bin/env python3
-#
-# Experimental implementation of asyncio support.
-#
-# This file is part of pySerial. https://github.com/pyserial/pyserial
-# (C) 2015 Chris Liechti <cliechti@gmx.net>
-#
-# SPDX-License-Identifier:    BSD-3-Clause
-"""\
-Support asyncio with serial ports. EXPERIMENTAL
-
-Posix platforms only, Python 3.4+ only.
-
```

Comment:
Well, I replaced most of the code in this file. You may want to have a look at the latest version here: https://github.com/m-labs/asyncserial